validator/README.md

```diff
@@ -123,3 +123,31 @@ $ python
 ```
 
 Now `cd amphtml/validator` and run `python build.py`.
+
+### Locally Reproduce Validator Tests of Circle CI workflow
+
+In the case the test passes in your local machine but fails in Circle CI,
+you can use docker to reproduce the test errors.
+
+1. Start an interacitve docker container
+    ```bash
+    docker run -it ubuntu bash
+    ```
+1. Run following commands in the container. Note that you are already the `root` user inside the docker container.
+    ```bash
+    apt update
+    apt install -y curl git gpg sudo
+    curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+    apt install -y nodejs
```

Comment:
Is it possible to simplify this via https://hub.docker.com/_/node?

Something like (untested):
```bash
docker run -it node:16 bash
```

and

```
apt install -y git
```

Reply:
Done